api/buildstatus/buildstatus_handler.go

```diff
@@ -0,0 +1,151 @@
+package buildstatus
+
+import (
+	"bytes"
+	"fmt"
+	"html/template"
+	"log"
+	"sort"
+	"strings"
+
+	"github.com/equinor/radix-api/api/utils"
+	"github.com/equinor/radix-api/models"
+	v1 "github.com/equinor/radix-operator/pkg/apis/radix/v1"
+	"github.com/marstr/guid"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const BUILD_STATUS_SVG_RELATIVE_PATH = "../../badges/build-status.svg"
```

Comment:
I get a "FileNotFound" error when using this path.
"./badges/build-status.svg" works.
Is it woking on you pc?